### ISSUE TITLE
ValueError: Cannot assign "269": "DataQualityCheck.organization" must be a "Organization" instance.

### ISSUE THREAD

**Issue by nllong** [SEED-platform/seed#1635]
https://sentry.io/nrel/seed-dev1-django/issues/559400272/

Data Quality Checks were hanging due to the wrong type of organization.

Reimport a file and make sure it passes.

```
ValueError: Cannot assign "269": "DataQualityCheck.organization" must be a "Organization" instance.
(11 additional frame(s) were not displayed)
...
  File "seed/data_importer/tasks.py", line 189, in finish_mapping
    import_file.id)
  File "seed/decorators.py", line 63, in _wrapped
    response = fn(import_file_pk, *args, **kwargs)
  File "seed/data_importer/tasks.py", line 515, in _data_quality_check
    DataQualityCheck.retrieve(organization_id)
  File "seed/models/data_quality.py", line 506, in retrieve
    dq, _ = DataQualityCheck.objects.get_or_create(organization=organization)
  File "seed/models/data_quality.py", line 523, in __init__
    super(DataQualityCheck, self).__init__(*args, **kwargs)

ValueError: Cannot assign "269": "DataQualityCheck.organization" must be a "Organization" instance.
```

**Comment by nllong** [comment]
This is the same as #1634

**Comment by RDmitchell** [comment]
I closed #1634 (although I don't think my test was really valid because I didn't keep the DQ state the same as before ... grrr ). So I'm closing this also.
